docs/handbook/mix_server.md

```diff
@@ -433,3 +433,15 @@ SEND_RATE 30
 ```
 SEND_BURST 4
 ```
+
+- `START_KAETZCHEN` - Start the Kaetzchen identified by capability:
```

Comment:
can we make the command easier to type and shorter? maybe like just "START"?

Reply:
STARTK? I'm not sure START is unambiguous enough.

Reply:
is there already another start? i think it's pretty unambiguous if we document it.

Reply:
Not yet, but we may want a graceful STOP command so that mixes can shutdown at end-of-epoch (and not publish another descriptor - so in that case STOP might be used for that purpose)